### ISSUE TITLE
Artwork

### ISSUE THREAD

**Issue by jaxxzer** [jaxxzer/Gravital#3]
We need some sprites for gas giants, comets, and stars.

**Comment by jlee139** [comment]
Uploaded new Assets we can use. Let me know if they should be changed and will change them accordingly. 
1. 3 new gas planets (plus one with all three planets in one png)
2. 3 new stars (plus one with all three stars in one png)
3. 1 comet - This one will most likely change. I don't like how it looks right now.

Have only changed the gas planet code to display planet2 but not anything else.

**Comment by jaxxzer** [comment]
Please keep track of sources for assets, and include them in the index.html

**Comment by jaxxzer** [comment]
I have added a smoke animation for the gas particles, that looks pretty good. The only issue is that when tinting the sprite to match the gas planet's color, the animation fails to update. It looks like this is a bug in 2.2.2 and has been fixed since: 
https://github.com/photonstorm/phaser/issues/1070
http://phaser.io/examples/v2/display/tint-sprite-frame
I will try updating phaser to 2.4.6 later today.
